### ISSUE TITLE
Missing version tags

### ISSUE THREAD

**Issue by avehtari** [stan-dev/rstan#992]
The last version tag in the repo is 2.19.2.
CRAN has 2.21.3. 
Commit f72e7d7f says `Bump version 2.26.8`, but there is no tag.

Could we have version tags for the new versions than 2.19.2, too?

ping @hsbadr @jgabry

**Comment by hsbadr** [comment]
@avehtari I don't want to tag versions from temporary branches (`experimental` and `StanHeaders_2.26`). I'm waiting for @bgoodri to decide that we can merge them into `develop` and `master`, respectively. I think Ben wants to confirm the path to CRAN 1st.

**Comment by avehtari** [comment]
I assume `master` has retired as the last commit there is more than 7 years ago.
I was hoping that there would be a tag for the version distributed via https://mc-stan.org/r-packages/

**Comment by hsbadr** [comment]
Makes sense. I've tagged the latest packaged version: https://github.com/stan-dev/rstan/releases/tag/v2.26.8. Note that this version is based on `StanHeaders_2.26` branch, which is patched to address backward compatibility issues at the C++ level, for CRAN. The development version is at `v2.29.1.9000` in the `experimental` branch. The two branches are diverged.

I hope that we will merge `experimental` into `develop` and tag releases from there. I defer to @bgoodri.

**Comment by hsbadr** [comment]
I've tagged all `2.26.x` versions.

**Comment by hsbadr** [comment]
@bgoodri I see that v2.21.5 has been submitted to CRAN. Could you push the changes and add a tag for it?
